projects/betafinance/index.js

```diff
@@ -1,59 +1,64 @@
-const ADDRESSES = require('../helper/coreAssets.json')
-const sdk = require("@defillama/sdk");
-const { getConfig } = require('../helper/cache')
-const { sumTokens } = require("../helper/unwrapLPs");
+const { getConfig } = require("../helper/cache");
 
-const config = {
-  ethereum: {
-    bank: '0x972a785b390D05123497169a04c72dE652493BE1',
-    collaterals: [
-      ADDRESSES.ethereum.USDT,
-      ADDRESSES.ethereum.USDC,
-      ADDRESSES.ethereum.DAI,
-      ADDRESSES.ethereum.WETH,
-    ],
-    poolURL: 'https://beta-reward-xvn33y7hlq-uc.a.run.app/beta_active_eth/reward_infos'
-  },
-  avax: {
-    bank: '0xf3a82ddd4fbf49a35eccf264997f82d40510f36b',
-    collaterals: [
-      ADDRESSES.avax.USDT_e,
-      ADDRESSES.avax.USDC_e,
-      ADDRESSES.avax.DAI,
-      ADDRESSES.avax.WETH_e,
-      ADDRESSES.avax.WAVAX,
-    ],
-    poolURL: 'https://beta-reward-xvn33y7hlq-uc.a.run.app/beta_active_avax/reward_infos'
-  }
-}
+const OMNI_LENS_ADDRESS = "0x4F003a858644E085186dfbC991E872b8B4Aac507";
 
-const underlyingABI = "address:underlying"
+const GET_MARKET_ADDRESS_URL =
```

Comment:
hi @WhatTheFar this endpoint seems to be down. also, instead of editing existing adapter file, I think it is better create a new folder for v2 and list it as "Beta Finance V2"?